src/modules/home/components/LoggedInHome/LoggedInHome.tsx

```diff
@@ -19,37 +21,82 @@ import { homeHeroRequests } from './__mocks__/HomeHeroRequests.mock';
 const LoggedInHome: FC = () => {
 	const { t } = useTranslation();
 	const [query, setQuery] = useQueryParams({
-		returnToRequestAccess: StringParam,
+		readingRoomId: StringParam,
 	});
 
 	const [isOpenRequestAccessBlade, setIsOpenRequestAccessBlade] = useState(false);
 
 	const user = useSelector(selectUser);
 
+	const { mutateAsync: createVisitRequest } = useCreateVisitRequest();
+
 	// Open request blade after user requested access and wasn't logged in
 	useEffect(() => {
-		if (query.returnToRequestAccess) {
+		if (query.readingRoomId) {
 			setIsOpenRequestAccessBlade(true);
 		}
-	}, [query.returnToRequestAccess]);
+	}, [query.readingRoomId]);
 
 	/**
 	 * Methods
 	 */
 
 	const onCloseRequestBlade = () => {
-		if (query.returnToRequestAccess) {
-			setQuery({ returnToRequestAccess: undefined });
+		if (query.readingRoomId) {
+			setQuery({ readingRoomId: undefined });
 		}
 		setIsOpenRequestAccessBlade(false);
 	};
 
-	const onRequestAccessSubmit = () => {
-		// TODO: add create request call here
-		setIsOpenRequestAccessBlade(false);
+	const onRequestAccessSubmit = async (values: RequestAccessFormState) => {
+		try {
+			if (!user) {
+				toastService.notify({
+					title: t('Je bent niet ingelogd'),
+					description: t('Je bent niet ingelogd, log opnieuw in en probeer opnieuw.'),
+				});
+				return;
+			}
+			if (!query.readingRoomId) {
+				toastService.notify({
+					title: t('Selecteer eerst een leeszaal'),
+					description: t(
+						'De leeszaal waarvoor je een aanvraag wil indienen is niet ingesteld.'
+					),
+				});
+				return;
+			}
```

Comment:
Kan vermeden worden door `RequestAccessBlade` te verbergen als:
`isOpenRequestAccessBlade && !!query.readingRoomId`

Reply:
Dan ziet de gebruiker niets van error message.
Is het niet beter om de gebruiker te informeren wat er precies mis gaat?
Deze approach zorgt ook dat de typescript compiler niet klaagt:
```
query.readingRoomId could be undefined, and undefined cannot be assigned to string
```